client/src/components/Experience.js

```diff
@@ -1,66 +1,79 @@
-import React, { useState } from "react";
+import React from "react";
 import { Form } from "semantic-ui-react";
 import { DateInput } from "semantic-ui-calendar-react";
 
-const Experience = ({ handleExperience, experienceData }) => {
-  const [experienceIndex] = useState(0);
-
+const Experience = ({
+  handleExperience,
+  experienceData,
+  addMoreExperience
+}) => {
   return (
     <Form>
```

Comment:
This <Form> is causing errors in the logs because you can't embed a Form inside another Form. Could you do the cleanup please?